src/game_interpreter_battle.cpp

```diff
@@ -602,95 +602,75 @@ bool Game_Interpreter_Battle::CommandManiacControlAtbGauge(lcf::rpg::EventComman
 		return true;
 	}
 
-	// 0 actor
-	// 1 party member
-	// 2 entire party
-	// 3 troop member
-	// 4 entire troop
-	int targetFlags = com.parameters[0];
-	// 0 constant
-	// 1 variable
-	// 2 variable id
-	int targetReferenceFlags = com.parameters[1];
-	int targetReferenceIdentifier = com.parameters[2];
-	// 0 set
-	// 1 add
-	// 2 sub
-	int operationFlags = com.parameters[3];
-	// 0 value
-	// 1 percentage
-	int operandFlags = com.parameters[4];
-	// 0 constant
-	// 1 variable
-	// 2 variable id
-	int valueReferenceFlags = com.parameters[5];
-	int valueReferenceIdentifier = com.parameters[6];
-
-	auto getVariableOrValue = [](int flags, int identifier) {
-		switch (flags) {
-			case 0:
-				return identifier;
-				break;
-			case 1:
-				return Main_Data::game_variables->Get(identifier);
-				break;
-			case 2:
-				return Main_Data::game_variables->Get(Main_Data::game_variables->Get(identifier));
-				break;
-		}
-	};
+	int target_flags = com.parameters[0];
+	int target_reference_flags = com.parameters[1];
+	int target_reference_identifier = com.parameters[2];
+	int operation_flags = com.parameters[3];
+	int operand_flags = com.parameters[4];
+	int value_reference_flags = com.parameters[5];
+	int value_reference_identifier = com.parameters[6];
 
-	int targetId = getVariableOrValue(targetReferenceFlags, targetReferenceIdentifier);
+	int target_id = ValueOrVariable(target_reference_flags, target_reference_identifier);
 
-	auto getAtbValue = [getVariableOrValue, valueReferenceFlags, valueReferenceIdentifier](int flags) {
-		int value = getVariableOrValue(valueReferenceFlags, valueReferenceIdentifier);
+	auto getAtbValue = [this, value_reference_flags, value_reference_identifier](int flags) {
+		int value = ValueOrVariable(value_reference_flags, value_reference_identifier);
 
 		switch (flags) {
 			case 0:
+				// value
 				return value;
 				break;
 			case 1:
+				// percentage
 				return (int) ((double)value / 100 * Game_Battler::GetMaxAtbGauge());
 				break;
```

Comment:
Can you add a ``default: return 0`` case to silence this compiler warning?

You can add that change to the previous commit: ``git commit --amend``, then force push